spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
+  let(:valid_user) do
```

Comment:
- **[OPTIONAL]** We can use `factory_bot` gem to handle creation of models instead of writing our own in every test.

Reply:
thank you! We will have it in mind for future projects